TEST_APPS/device/socket_app/mbed_app.json

```diff
@@ -7,6 +7,7 @@
     ],
     "target_overrides": {
         "*": {
+            "target.features_add": ["LWIP", "COMMON_PAL"],
```

Comment:
This one could be still in, not reverting?

Reply:
@bridadan Was this one that was missed from the original PR?